optuna_dashboard/ts/components/PreferenceHistory.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { FC, useState } from "react"
+import {
+  Typography,
+  Box,
+  useTheme,
+  Card,
+  CardContent,
+  CardActions,
+} from "@mui/material"
+import ClearIcon from "@mui/icons-material/Clear"
+import IconButton from "@mui/material/IconButton"
+import OpenInFullIcon from "@mui/icons-material/OpenInFull"
+import Modal from "@mui/material/Modal"
+import { red } from "@mui/material/colors"
+
+import { TrialListDetail } from "./TrialList"
+import { MarkdownRenderer } from "./Note"
+import { formatDate } from "../dateUtil"
+
+type TrialType = "worst" | "none"
+
+const CandidateTrial: FC<{
+  trial: Trial
+  type: TrialType
+}> = ({ trial, type }) => {
+  const theme = useTheme()
+  const trialWidth = 500
```

Comment:
Does it sound reasonable to make the width smaller?

```suggestion
  const trialWidth = 300
```